src/main/java/org/tugraz/sysds/parser/DMLTranslator.java

```diff
@@ -1100,7 +1100,7 @@ public void constructHops(StatementBlock sb) {
 
 						if (source instanceof BuiltinFunctionExpression){
 							BuiltinFunctionExpression BuiltinSource = (BuiltinFunctionExpression)source;
-							if (BuiltinSource.getOpCode() == Builtins.TIME)
+							if (BuiltinSource.getOpCode() == Builtins.TIME || BuiltinSource.getOpCode() == Builtins.TYPEOF  )
```

Comment:
Why should we split DAGs for typeof. For time its just to ensure it really measures the specified code, but for typeof only data dependencies should matter, right?